newm-server/src/main/kotlin/io/newm/server/features/earnings/repo/EarningsRepositoryImpl.kt

```diff
@@ -312,4 +323,62 @@ class EarningsRepositoryImpl(
 
         schedulerDaemon.scheduleJob(jobDetail, trigger)
     }
+
+    override suspend fun generateRetrieveEarningsPaymentTransaction(
+        songId: SongId,
+        requesterId: UserId,
+        sourceUtxos: List<Utxo>,
+        changeAddress: String
+    ): String {
+        log.debug { "generateRetrieveEarningsPaymentTransaction: songId = $songId" }
+
+        checkRequester(songId, requesterId, verified = true)
```

Comment:
This line needs to be deleted. This is for distribution where we need to make sure that the person requesting the song minting is the owner of the song. For Earnings, "anybody" can retrieve earnings for any song.